client/src/pages/Landing.tsx

```diff
@@ -1,40 +1,32 @@
-/* eslint-disable react/jsx-props-no-spreading */
-import React from 'react';
-import { useSelector, useDispatch, shallowEqual } from 'react-redux';
-import { AppDispatch, RootState } from '../store';
-import { StyledForm, StyledLanding } from '../styles/Stylesheet';
-import UserInfoForm from '../components/UserInfoForm';
-import UserPrefsForm from '../components/UserPrefsForm';
-import { setIsOpen } from '../store/FormSlice';
+import React, { useCallback, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { StyledLanding } from '../styles/Stylesheet';
 import logo from '../img/logo2.png';
+import { setUsersData } from '../store/DataSlice';
 
 function Landing() {
-  const dispatch: AppDispatch = useDispatch();
-  const { isOpen, nextForm } = useSelector(
-    (state: RootState) => state.form,
-    shallowEqual,
-  );
+  const dispatch = useDispatch();
+  const getData = useCallback(async () => {
+    const res = await fetch('/users');
+    const data = await res.json();
+    dispatch(setUsersData(data));
+  }, [dispatch]);
+  useEffect(() => {
```

Comment:
Be sure to be consistent re: line breaks (or lack thereof) between blocks of code... your linter/prettier can help, too!